exonum/src/proto/push_proto_files.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```

Comment:
Should this script be a part of core repository?

Reply:
Good point 👍🏻

Reply:
According to the discussion in the [ECR-3472](https://jira.bf.local/browse/ECR-3472) if we consider the core repo a single source of changes for proto files this looks more reasonable - the core team knows when the proto files format gets stabilized and when it's time to update it for others (without exposing intermediate API to the public for example).

Reply:
As for me, it makes more sense to have this script in the repo with updated proto files and invoke it on each release, so this script will extract proto files from `exonum`, commit it to the exported-protos repo and create a tag denoting the release of `exonum` for which files were extracted.

Reply:
Well, makes sense if proto files remain within the single repo.